pkg/plugins/plugindef/plugindef.cue

```diff
@@ -364,6 +364,9 @@ schemas: [{
 			reqSignedIn?: bool
 			reqRole?:     string
 
+			// RBAC action the user must have to access the route
+			action?: string
```

Comment:
Do we prefer `reqAction` to be consistent with `reqRole` or `action` to be consistent with the includes' `action` field?

Reply:
I'd go with `reqAction` to make it clear it's a requirement. Also an example will help for people to understand what this does.

Unrelated but does this support multiple actions or only one?

Reply:
Ok :ok_hand:

> Also an example will help for people to understand what this does.
What do you have in mind? Plugin example?

It supports only one action. We'll see if we ever need to support something more complex.

Reply:
> What do you have in mind? Plugin example? Or plugin-tools docs update with an example?
For this case, I only mean in the comment: `// RBAC action the user must have to access the route i.e. plugin-id.projects:read`

> It supports only one action
:+1: